### DISCUSSION TITLE
Virtual keyboard

### DISCUSSION THREAD

**Discussion by Taifuuni** [Gulivertx/cbatmo#53] [General]
I suggest to add a virtual keybord that you can use with the 7" Raspberry touch screen. One solution is this https://github.com/Mottie/Keyboard which is implemented in the webpage. No need for dependencies.

**Comment by Gulivertx** [comment]
Thanks for your proposal but there was already a virtual keyboard in previous versions, but I removed it because from my point of view it has to be part of OS and not from the app. I would not put back a virtual keyboard myself or accept pull request to add it.
And I mainly not want a Jquery version which is not an option for this app has there is not any Jquery dependency (JQuery is from my point of view end of its life for modern web app).

If you want you can Fork this project to add it and use the old code to add it for your use. Please have a look at this if you have any interest :
![Virtual keyboard](https://github.com/Gulivertx/cbatmo/blob/master/screenshots/screenshot_002.png?raw=true "Virtual keyboard")

https://github.com/Gulivertx/cbatmo/blob/208da2b2422b2f1d7c8627506a625e6315de12fb/client/components/VirtualKeyboardInput.js
